google/cloud/spanner_dbapi/cursor.py

```diff
@@ -207,25 +238,26 @@ def execute(self, sql, args=None):
             # self._run_prior_DDL_statements()
             self.connection.run_prior_DDL_statements()
 
-            if not self.connection.autocommit:
```

Comment:
Moving several commands away of this `if` statement to avoid duplicating them further